compiler/modules/front.syntax/token.bal

```diff
@@ -564,6 +564,15 @@ public readonly class SourceFile {
         return { startPos, endPos: createPosition(lineNum, endColumnNum) };
     }
 
+    public function startPos() returns Position {
+        return createPosition(1, 0);
+    }
+
+    public function endPos() returns Position {
+        int lastLine = self.lines.length() + 1;
```

Comment:
Why isn't the endPos the end of the last line?

Reply:
fixed